cmake/iotjs.cmake

```diff
@@ -553,7 +556,13 @@ target_link_libraries(${TARGET_LIB_IOTJS}
   ${MBEDTLS_LIBS}
   ${EXTERNAL_LIBS}
 )
-
+target_link_libraries(${TARGET_STATIC_IOTJS}
```

Comment:
This isn't necessary, the library will already be linked by the previous link_libraries call, since `TARGET_LIB_IOTJS` and  `TARGET_STATIC_IOTJS` are just an alias for the same `libiotjs` target.

Reply:
ok, I just verified